src/Jsontracer.ts

```diff
@@ -0,0 +1,142 @@
+/*
+ * Copyright (c) 2021 Samsung Electronics Co., Ltd. All Rights Reserved
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import * as fs from 'fs';
+import * as vscode from 'vscode';
+import {getNonce} from './Config/GetNonce';
+import {getWebviewOptions} from './Jsontracer/GetWebviewOptions';
+
+export class Jsontracer {
+  /**
+   * Track the current panel. Only allow a single panel to exist at a time.
+   */
+  public static currentPanel: Jsontracer|undefined;
+
+  private static readonly viewType = 'Jsontracer';
+
+  private readonly _panel: vscode.WebviewPanel;
+  private readonly _extensionUri: vscode.Uri;
+  private _disposables: vscode.Disposable[] = [];
+
+  public static createOrShow(extensionUri: vscode.Uri) {
+    const column =
+        vscode.window.activeTextEditor ? vscode.window.activeTextEditor.viewColumn : undefined;
+
+    // If we already have a panel, show it.
+    if (Jsontracer.currentPanel) {
+      Jsontracer.currentPanel._panel.reveal(column);
+      return;
+    }
+
+    // Otherwise, create a new panel.
+    const panel = vscode.window.createWebviewPanel(
+        Jsontracer.viewType,
+        'Jsontracer',
```

Comment:
IIUC, this two values are same.
Can I ask you why different variables are used for same value?

Reply:
In this case, the same value was used, but the meaning is different.
first parameter means identifies the type of the webview, used internally. second parameter is title of the panel displayed to the user.